sphinx/install.rst

```diff
@@ -8,14 +8,14 @@ Install
 First of all, we recommend using a `virtual environment (venv) <https://docs.python.org/3/tutorial/venv.html>`_.
 You need ``python>=3.9``.
 
-Then, you can install ``skore`` by using ``pip``:
+Then, you can install skore by using ``pip``:
 
 .. code:: console
 
     pip install -U skore
```

Comment:
Do we need to keep the -U ? It's not required by default to install, I think it adds complexity. I have the impression that I rarely see this option suggested.

Reply:
scikit-learn does that, it is to update the package if needed